ninja/src/test/java/controllers/ApiControllerDocTesterTest.java

```diff
@@ -0,0 +1,27 @@
+package controllers;
+
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.junit.Assert.assertThat;
+import org.doctester.testbrowser.Request;
+import org.doctester.testbrowser.Response;
+import org.junit.Test;
+import ninja.NinjaDocTester;
+
+public class ApiControllerDocTesterTest extends NinjaDocTester {
```

Comment:
question: is there a reason not to follow the naming convention for test classes, i.e. the class names should end in "UnitTest" or "IntegrationTest", as appropriate

Reply:
Fixed to ApiControllerDocTesterUnitTest
Ninja only has unit test (even for integration test) and Test class is created by the framework be default.